sklearnex/__init__.py

```diff
@@ -29,6 +29,16 @@
     'svm', 'unpatch_sklearn', 'utils'
 ]
 
+try:
+    import onedal._onedal_py_dpc as _backend
+    _is_dpc_backend = True
+except ImportError:
+    import onedal._onedal_py_host as _backend
+    _is_dpc_backend = False

```

Comment:
no need to copy past code from onedal. Just reuse `_is_dpc_backend` variable from onedal.

Reply:
Fixed.